Dockerfile

```diff
@@ -93,7 +93,7 @@ RUN useradd --create-home --shell /bin/bash ${USER}
 # with sudo access and no password
 RUN usermod -append --groups sudo ${USER}
 RUN echo "%sudo ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
-RUN pip install --quiet --upgrade Cython==0.28.6
+RUN pip install --quiet --upgrade cython==0.28.6
```

Comment:
Minor, why changing the case, it just create "noise" in the history?

Reply:
I didn't, you did, and I rebased on your changes :p

Reply:
https://github.com/kivy/python-for-android/commit/19d2de09f3bd42e910ef631c6d111f58507dd4fa#diff-3254677a7917c6c01f55212f86c57fbfR96
In my changes it's with a capital plus the version change. In your changes you're only updating the case.
It's minor anyway, but I think it could be avoided